packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Badge from '../../Badge';
+
+/**
+ * Cell renderer that displays a badge
+ */
+function CellBadge({ cellData, rowIndex }) {
+	return <Badge id={`${rowIndex}`} label={cellData} selected />;
```

Comment:
shouldn't id be prefixed ? tc-list-badge-id or something like that

Reply:
The badge component add a prefix, passing the line nb outputs `tc-badge-select-4`